test/test_encryption.py

```diff
@@ -2146,6 +2145,14 @@ def test_04_kmip(self):
         ):
             self.client_encryption_invalid_hostname.create_data_key("kmip")
 
+    def test_05_tlsDisableOCSPEndpointCheck_is_permitted(self):
+        providers = {"aws": {"accessKeyId": "foo", "secretAccessKey": "bar"}}
+        options: dict = dict(aws=dict(tlsdisableocspendpointcheck=True))
```

Comment:
Can this be:
```python
options = {"aws": "tlsDisableOCSPEndpointCheck": True}
```

so that it's consistent with "providers"?

Reply:
Done